packages/client/_src/Resources/Blog.ts

```diff
@@ -0,0 +1,6 @@
+// codegen:start {preset: barrel, include: ./Blog/*.ts, export: { as: 'PascalCase' }, nodir: false }
```

Comment:
Is this codegeneration automatically handled by eslint or do I need to set up my ide in some special way? Is there a place I can check how this is handled?

Reply:
Handled by eslint